crates/ruff_python_index/src/indexer.rs

```diff
@@ -68,18 +72,33 @@ impl Indexer {
                 Tok::Newline | Tok::NonLogicalNewline => {
                     line_start = range.end();
                 }
+                Tok::FStringStart => {
+                    f_string_start_count += 1;
+                    if f_string_start_count == 1 {
+                        first_f_string_start_range = Some(*range);
+                    }
+                }
+                Tok::FStringEnd => {
+                    // This is always going to be > 0, because the lexer will only
+                    // emit the end token if there was a start token to begin with.
+                    f_string_start_count -= 1;
+                    if f_string_start_count == 0 {
+                        if let Some(start_range) = first_f_string_start_range.take() {
+                            let f_string_range = TextRange::new(start_range.start(), range.end());
+                            f_string_ranges.push(f_string_range);
+
+                            if matches!(locator.slice(range), "'''" | r#"""""#) {
+                                triple_quoted_string_ranges.push(f_string_range);
+                            }
```

Comment:
In previous implementation, triple-quoted f-strings were only stored in `triple_quoted_string_ranges` and not in `f_string_ranges`. Was that an expected behavior?

Currently, I'm storing in both ranges but I can revert back to the original bahavior.